ingestion/src/metadata/ingestion/source/dashboard/dashboard_service.py

```diff
@@ -550,28 +550,6 @@ def test_connection(self) -> None:
     def prepare(self):
         """By default, nothing to prepare"""
 
-    def fqn_from_context(self, stage: NodeStage, entity_name: C) -> str:
```

Comment:
After the changes to topology context in 1.4.x this method is not longer being overridden and the name of the method has changed to `fqn_from_stage`.
We'll remove the method from here and add the one liner logic directly in Topology Context to keep it centralized.